src/node_api.cc

```diff
@@ -2017,8 +2017,8 @@ napi_status napi_wrap(napi_env env,
 
   // Insert the wrapper into the object's prototype chain.
   v8::Local<v8::Value> proto = obj->GetPrototype();
-  wrapper->SetPrototype(proto);
-  obj->SetPrototype(wrapper);
+  CHECK(wrapper->SetPrototype(context, proto).ToChecked());
+  CHECK(obj->SetPrototype(context, wrapper).ToChecked());
```

Comment:
nit: `FromJust()` is used in this file but we are not consistent elsewhere so I don't mind.